utils/pkb_automated_runner.sh

```diff
@@ -18,32 +18,36 @@ function run_pkb_provision() {
 function run_pkb_benchmark() {
     local file="$1"
     local unique_run_id="$2"
+    local subdir="$3"
     python "$PKB_PATH"/pkb.py --benchmarks=fio --run_uri="$unique_run_id" \
       --num_vms=1 --benchmark_config_file=pkb_fio_flags.yaml \
       --fio_jobfile="$file" --run_stage=run,cleanup,teardown
-    cp -r /tmp/perfkitbenchmarker/runs/"$unique_run_id" "$OUT_DIRECTORY"
+    if [[ ! -d "$OUT_DIRECTORY/$subdir" ]]; then
+        mkdir -p "$OUT_DIRECTORY/$subdir"
+    fi
+    cp -r /tmp/perfkitbenchmarker/runs/"$unique_run_id" "$OUT_DIRECTORY/$subdir"
 }
 
-
 if [ $# -lt 3 ]; then
   echo "Too few arguments supplied, should be in format [vms, output directory, config file]"
   exit -1
 fi
 
 if [[ ! -d "$OUT_DIRECTORY" ]]; then
-    mkdir -p "$OUT_DIRECTORY"
+  mkdir -p "$OUT_DIRECTORY"
```

Comment:
Good point.  With the "-p" flag, we only need the one in the run_pkb_benchmark() function.  Also, I confirmed that $OUT_DIRECTORY isn't needed before the directory check at line 25.  

Result: I will remove the directory check and make at lines 36-38